lib/telegraf/telegraf.conf.erb

```diff
@@ -562,7 +562,7 @@
 #   servers = ["127.0.0.1:27017"]
 
 
-<% config_file['mysql'].each do |server_url| %>
+<% (config_file['mysql'] || []).each do |server_url| %>
```

Comment:
Refactoring the configuration code in this way seemed outside the bounds of this bug fix.